.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: 'Bug report'
+about: Template for reporting a bug
+title: 'fix: [bug description]'
+labels: bug
+assignees: ''
+
+---
+
+### Description
+
+Add a short description of the problem.
+
+### To reproduce
+
+Steps to reproduce the behaviour:
+1. scan using these flags/this config `...`
+2. on this file `...`
+3. `...`
+
+Expected behabiour:
+Actual behaviour:
+
+### Version/platform info
+
+Version of CVE-bin-tool:
```

Comment:
For this type of debug info request, I've found more people will fill it out if you give a command (or commands) to copy and paste like:

Python version (`python3 --version`):
Operating system string (Linux users, run `uname -a`, Windows users run `systeminfo | findstr /B /C:"OS Name" /C:"OS Version"`):

Reply:
These are excellent suggestions; let me go get them added.